src/core/action_descriptor.ts

```diff
@@ -1,6 +1,8 @@
+import { ExtendedAddEventListenerOptions } from "../extensions"
+
 export interface ActionDescriptor {
   eventTarget: EventTarget
-  eventOptions: AddEventListenerOptions
+  eventOptions: ExtendedAddEventListenerOptions
```

Comment:
One suggestion as we now have properties that are not anymore linked to the EventListner maybe we could name that concept `EventModifiers`

That would be also inlined with the naming conventions in Vue 3 https://vuejs.org/guide/essentials/event-handling.html#event-modifiers

Reply:
@adrienpoly Resolved in ec703c4e359faa51313441092bd001bfbb632cbb. 🚀